src/dstack/_internal/server/utils/logging.py

```diff
@@ -18,25 +19,28 @@ def filter(self, record: logging.LogRecord) -> bool:
 
 
 def configure_logging():
-    default_formatter = "standard"
-    formatters = {
-        default_formatter: logging.Formatter(
-            fmt="%(levelname)s %(asctime)s.%(msecs)03d %(name)s %(message)s",
-            datefmt="%Y-%m-%dT%H:%M:%S",
-        ),
-        "json": jsonlogger.JsonFormatter(
-            "%(asctime)s %(name)s %(levelname)s %(message)s",
-            json_ensure_ascii=False,
-            rename_fields={"name": "logger", "asctime": "timestamp"},
-        ),
-    }
+    default_formatter = "rich"
     log_format = os.getenv("DSTACK_SERVER_LOG_FORMAT", default_formatter)
-    formatter = formatters.get(log_format, formatters[default_formatter])
-
-    root_logger = logging.getLogger(None)
-    handler = logging.StreamHandler(stream=sys.stdout)
-    handler.addFilter(AsyncioCancelledErrorFilter())
+    if log_format == "rich":
+        formatter = logging.Formatter(fmt="%(message)s", datefmt="[%X]")
+        handler = DstackRichHandler(console=console)
+    else:
+        formatters = {
```

Comment:
(nit) Looks a bit too complicated because if-else and the dict are used for the same purpose. Consider
```python
formatters = {
    "standard": ...,
    "rich": ...,
    "json": ...,
}
handlers = {
    "standard": ...,
    "rich": ...,
    "json": ...,
}
formatter = formatters.get(log_format, formatters[default_format])
handler = handlers.get(log_format, handlers[default_format])
```